fileattrs/modulesload.attr

```diff
@@ -0,0 +1,2 @@
+%__modulesload_requires		%{_rpmconfigdir}/find-requires.ksyms --tumbleweed %{?sle_version:0}%{!?sle_version:1}
+%__modulesload_path		^(/usr/lib|/etc)/modules-load.d/.*\.conf$
```

Comment:
We should not be creating dependencies in `/etc/modules-load.d`, since that's locally managed by the admin.

Reply:
You are right, but there are still packages that do that, and I see no reason not to generate the dependencies for them.

Reply:
I do: I want those to be invisible until they fix themselves to do the right thing.

Reply:
I wasn't aware that it's up to you, personally, to decide about this sort of policy.

Reply:
But I guess I can live with this for Factory.